sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,724 @@
+
+```ts
+// use the design token to build styles
+const styles = css`
+  :host {
+    background: ${specialColor};
+  }
+`;
+```
+
+**Example: [Setting values](https://www.fast.design/docs/design-systems/design-tokens#setting-values)**
+```ts
+// change the value for a given element
+specialColor.setValueFor(ancestor, "#FFF");
+specialColor.setValueFor(descendent, "#000");
+```
+
+**Example: [Getting values](https://www.fast.design/docs/design-systems/design-tokens#getting-values)**
+```ts
+// get the value
+specialColor.getValueFor(ancestor); // "#FFF"
+specialColor.getValueFor(descendent); // "#000"
+```
+
+**Example: [Deleting values](https://www.fast.design/docs/design-systems/design-tokens#deleting-values)**
+```ts
+// unset the value (inherits from ancestor)
+specialColor.deleteValueFor(descendent);
+specialColor.getValueFor(descendent); // "#FFF"
+```
+
+---
+
+#### Configuring a FAST design token
+
+FAST exposes non-color related [Design Tokens](https://www.fast.design/docs/design-systems/fast-frame#fast-frame-design-tokens) that can be used to configure components stylistically.  For Design Tokens related to color, see the [adaptive color system](https://www.fast.design/docs/design-systems/fast-frame/#adaptive-color-system).
+
+**Example: Changing the value for the `typeRampBaseFontSize` design token**
+
+```ts
+import { typeRampBaseFontSize } from "@microsoft/fast-components";
+import { FASTElement } from '@microsoft/fast-element';
+
+const myElement = document.querySelector("my-element") as FASTElement;
+
+typeRampBaseFontSize.setValueFor(myElement, "20px");
+```
+
+---
+
+## [Contributing to FAST](https://www.fast.design/docs/community/join)
+
+**Connect with us**:
+
+- Join our [Discord](https://discord.gg/FcSNfg4) server.
+- Report bugs, request features through [Github](https://github.com/Microsoft/fast/issues/new/choose).
+
+**Unsure of what to work on?**
+
+- Here are [good first issues](https://github.com/Microsoft/fast/labels/community:good-first-issue).
+
+---
+### [Contributor guide](https://www.fast.design/docs/community/contributor-guide/)
+- [Machine setup](https://www.fast.design/docs/community/contributor-guide/#machine-setup)
+- [Cloning the repository](https://www.fast.design/docs/community/contributor-guide/#cloning-the-repository)
+- [Installing & building](https://www.fast.design/docs/community/contributor-guide/#installing-and-building)
+- [Testing](https://www.fast.design/docs/community/contributor-guide/#testing)
+- [Submitting a pull request](https://www.fast.design/docs/community/contributor-guide/#submitting-a-pull-request)
+- [Merging a pull request](https://www.fast.design/docs/community/contributor-guide/#merging-a-pull-request)
+
+---
+
+### [Contributing to `fast-component`](https://www.fast.design/docs/community/contributor-guide)
+
+```shell
+cd packages/web-components/fast-components
+yarn start
```

Comment:
alternatively, `yarn workspace @microsoft/fast-components start`

Reply:
@radium-v Thank you

@EisenbergEffect What is your preference?

The docs have `yarn start`

![Screen Shot 2021-12-16 at 1 38 28 PM](https://user-images.githubusercontent.com/35910788/146452925-b3281e91-e2f1-4023-be4d-a3065d5e1665.png)

Reply:
Let's leave it as is for now so I can get this merged and published before heading out for vacation 😄